sqlx-core/src/postgres/types/ltree.rs

```diff
@@ -0,0 +1,72 @@
+use std::fmt::{self, Display, Formatter};
+use std::str::FromStr;
+use crate::decode::Decode;
+use crate::encode::{Encode, IsNull};
+use crate::error::{Error, BoxDynError};
+use crate::postgres::{PgArgumentBuffer, PgValueFormat, PgTypeInfo, PgValueRef, Postgres};
+use crate::types::Type;
+
+#[derive(Clone, Debug, Default, PartialEq)]
+pub struct PgLTree {
```

Comment:
I'm wondering if should push this as separate crate. It seems it could ease the work that i want to put in sea-orm

Reply:
You could reexport it in SeaORM.

Reply:
Yes but currently sea-query for example has no dependency to sqlx itself